extensions/Scribunto/tests/engines/LuaCommon/TitleLibraryTest.php

```diff
@@ -34,7 +34,6 @@ function setUp() {
 			$data = array(
 				'iw_prefix' => 'scribuntotitletest',
 				'iw_url'    => '//test.wikipedia.org/wiki/$1',
-				'iw_wikiid' => 0,
```

Comment:
The same here :)

https://www.mediawiki.org/wiki/Manual:Interwiki_table#iw_wikiid - I'd rather try to avoid messing with core MediaWiki tables. Let's alter the column definition instead of removing it.